libraries/chain/include/eosio/chain/exceptions.hpp

```diff
@@ -322,6 +322,10 @@ namespace eosio { namespace chain {
                                     3060004, "Contract Query Exception" )
       FC_DECLARE_DERIVED_EXCEPTION( bad_database_version_exception, database_exception,
                                     3060005, "Database is an unknown or unsupported version" )
+      FC_DECLARE_DERIVED_EXCEPTION( database_revision_mismatch_exception, database_exception,
+                                    3060006, "Chainbase and chain-kv databases are at different revisions" )
+      FC_DECLARE_DERIVED_EXCEPTION( database_move_kv_disk_exception, database_exception,
```

Comment:
This exception and logic tied to it, can be removed since there will be no moving of RAM to DISK